docs/src/main/asciidoc/sending-emails.adoc

```diff
@@ -246,6 +246,36 @@ class MailTest {
 }
 ----
 
+== Gmail specific configuration
+
+If you want to use the gmail smtp server, first, please create a dedicated password in `google account > security > application password` or go to https://myaccount.google.com/apppasswords 
+
+When it done you can use it in your configuration. 
+
+With TLS:
+
+[source]
+----
+quarkus.mailer.from=YOUREMAIL@gmail.com
+quarkus.mailer.host=smtp.gmail.com
+quarkus.mailer.port=587
+quarkus.mailer.start-tls=REQUIRED
+quarkus.mailer.username=YOUREMAIL@gmail.com
+quarkus.mailer.password=YOURGENERATEDAPPLICATIONPASSWORD
+----
+
+Or with SSL:
+
+[source]
+----
+quarkus.mailer.from=YOUREMAIL@gmail.com
+quarkus.mailer.host=smtp.gmail.com
+quarkus.mailer.port=465
+quarkus.mailer.ssl=true
```

Comment:
this will work only in combination with `quarkus.ssl.native=true` .

Reply:
I can add it in the doc but I propose to change `SslNativeConfigBuildItem` as a `MultiBuildItem` and produce them in the `MailerProcessor` if the `MailConfig.ssl` is enable ? Are you ok @gsmet @machi1990  ?
Else I just document it....

Reply:
I added a note about the SSL support and will merge.

I wonder if we should enable SSL by default when the Mailer extension is used. @cescoffier WDYT?